dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -37,26 +37,32 @@
 		i2c-smb-4 = &i2c_smb_4;
 	};
 
-	pinmux {
+	pinmux: pinmux {
```

Comment:
If this node's path in the devicetree (here `/pinmux`) is always going to be the same, you can use `DT_PATH(pinmux)` in C instead of having to define the node label. Then you can remove this change. But adding a node label here is also fine.

Reply:
@mbolivar Thanks for the infomation.  I will keep it as is with the node label.  PINCTRL will cause most of this to change.  It appears almost all other SoC have moved to a partial PINCTRL implementation.